packages/mermaid/src/dagre-wrapper/index.js

```diff
@@ -13,6 +13,9 @@ import { insertNode, positionNode, clear as clearNodes, setNodeElem } from './no
 import { insertCluster, clear as clearClusters } from './clusters.js';
 import { insertEdgeLabel, positionEdgeLabel, insertEdge, clear as clearEdges } from './edges.js';
 import { log } from '../logger.js';
+import { getSubGraphTitleMargins } from '../utils/getSubGraphTitleMargins.js';
+
+const { subGraphTitleTotalMargin } = getSubGraphTitleMargins();
```

Comment:
This should be moved into the function, otherwise it'll cause bugs when multiple diagrams are rendered.

Reply:
The reason I put this here was because the recursive calls would keep calling `getSubGraphTitleMargins()` unnecessarily. But if it causes bugs, then I think there's no option, I'll put it back in.

Reply:
If you add the config as parameter, then that call shouldn't be a big issue.

Reply:
Done.